src/util.h

```diff
@@ -156,9 +156,10 @@ std::string GetArg(const std::string& strArg, const std::string& strDefault);
  *
  * @param strArg Argument to get (e.g. "-foo")
  * @param default (e.g. 1)
+ * @param novalue What to return if option provided with no value
  * @return command-line argument (0 if invalid number) or default value
  */
-int64_t GetArg(const std::string& strArg, int64_t nDefault);
+int64_t GetArg(const std::string& strArg, int64_t nDefault, int64_t nNoValue = 0);
```

Comment:
The exact same behavior can be achieved on init without touching this function.
I repeat the question, why not just return nDefault when the option is provided with no value?
In the concrete case of this PR, seems just a hack so that "-replacebyfee" is equivalent to "-replacebyfee=1" instead of "-replacebyfee=2" like it would be if this function received no extra parameters. In my opinion, that doesn't justify adding a third parameter to this function.
Anyway, if we switch to strings as we should, this discussion would be irrelevant (unless you want not using policy to default to one value, but using an empty -policy to default to another value).